Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -162,14 +162,13 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	 */
 	UFUNCTION(BlueprintCallable, Category= "C++")
 	void OnTakeActorsFromPoolCompleted(const TArray<FPoolObjectData>& CreatedObjects);
-
-	/** Updates star actor to locked/unlocked according to input amounnt
+	
+	/** Adds/Updates star actors
 	 * @param CreatedData Object received from Pool Manager which contains the reference to Start Widget 
-	 * @param AmountOfUnlockedPoints The number of stars (unlocked-stars as actors) to be added on top of the character
-	 * @param AmountOfLockedPoints The number of stars (locked-stars as actors) to be added on top of the character
+	 * @param AmountOfStars The number of stars to be added on top of the character
 	 */
 	UFUNCTION(BlueprintCallable, Category= "C++", meta = (BlueprintProtected))
-	void UpdateStarActor(const FPoolObjectData& CreatedData, float AmountOfUnlockedStars, float AmountOfLockedStars);
+	void SetOrUpdateStarActorMesh(UStaticMeshComponent* MeshComponent, float AmountOfStars, bool bIsLockedStar);
```

Comment:
Please completely update description of `SetOrUpdateStarActorMesh`-function:
- Missing description of MeshComponent and bIsLockedStar parameters
- Unnecessary description of removed CreatedData parameter
- UStaticMeshComponent should be with `class-forward

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/17/commits/cba3bdc6da66a863f46d6637e595031db53760af